src/app/space/create/pipelines/pipelines.component.html

```diff
@@ -1,36 +1,69 @@
 <div id="pipelines" class="">
   <div class="toolbar-header-pipelines">
-    <pfng-toolbar [config]="toolbarConfig"
-                  [viewTemplate]="addOpenShiftConsoleLink"
-                  (onFilterChange)="filterChange($event)"
-                  (onSortChange)="sortChange($event)">
-      <ng-template #addOpenShiftConsoleLink>
-        <div class="openshift-links" placement="bottom">
-          <div class='pull-right dropdown-kebab-pf dropdown' dropdown>
-            <button class='btn btn-link pull-right dropdown-toggle' type='button'
-                    aria-haspopup='true' aria-expanded='true' dropdownToggle>
-              <span class='fa fa-ellipsis-v'></span>
-            </button>
-            <ul class='dropdown-menu-right dropdown-menu' aria-labelledby='dropdownKebabRight9' role="menu" *dropdownMenu>
-              <li>
-                <a (click)="openForgeWizard(updateSpace)">Add to Space</a>
-              </li>
-              <li class="divider"></li>
-              <li [ngClass]="{'disabled': !consoleAvailable}">
-                <a [href]="openshiftConsoleUrl" *ngIf="consoleAvailable; else consoleDisabled" target="_blank">
-                  Open in OpenShift Console
-                </a>
-                <ng-template #consoleDisabled>
-                  <a tooltip="Openshift Console unavailable" target="_blank" disabled>
+    <f8-feature-toggle featureName="AppLauncher">
+      <pfng-toolbar [config]="toolbarConfig"
+                    [viewTemplate]="addOpenShiftConsoleLink"
+                    (onFilterChange)="filterChange($event)"
+                    (onSortChange)="sortChange($event)" default-level>
+        <ng-template #addOpenShiftConsoleLink>
+          <div class="openshift-links" placement="bottom">
+            <div class='pull-right dropdown-kebab-pf dropdown' dropdown>
+              <button class='btn btn-link pull-right dropdown-toggle' type='button'
+                      aria-haspopup='true' aria-expanded='true' dropdownToggle>
+                <span class='fa fa-ellipsis-v'></span>
+              </button>
+              <ul class='dropdown-menu-right dropdown-menu' aria-labelledby='dropdownKebabRight9' role="menu" *dropdownMenu>
+                <li>
+                  <a id="appWizardAnchor" (click)="openForgeWizard(updateSpace)">Add to Space</a>
```

Comment:
Why not just wrap this one link, with the AppLauncher feature flag, instead of the entire toolbar? That would allow us to add other feature flags to the dropdown if necessary.

Reply:
Sure Dan, can wrap just the anchor. It'll affect pf styles here so custom styles need to be handled in this case.